crates/bevy_render/src/shader/shader.rs

```diff
@@ -42,7 +50,7 @@ impl Into<shaderc::ShaderKind> for ShaderStage {
     }
 }
 
-#[cfg(target_os = "ios")]
+#[cfg(any(target_os = "ios", all(target_os = "macos", target_arch = "aarch64")))]
```

Comment:
The function below was updated recently to return a `Result`. You can copy what I added here to make this work on Apple Silicon:

https://github.com/bevyengine/bevy/pull/1027

I created my pull request before I saw yours, oops